controllers/horizon_controller.go

```diff
@@ -474,7 +474,7 @@ func (r *HorizonReconciler) generateServiceConfigMaps(
 	h *helper.Helper,
 	envVars *map[string]env.Setter,
 ) error {
-	var memcachedSvc string
+	var podList []string
```

Comment:
memcachedHostList

Reply:
I feel conflicted about this. It makes sense from the memcached perspective, but in terms of Kubernetes using HostList seems to imply I'm getting information about the actual OpenShift nodes rather than a pod. Maybe I could use a longer variable name to clarify exactly what this is? Something like `podFqdnList` or something? `podHostnameList`?